level3/[level3] 이도형/java-racing-car/src/main/java/Model/GameModel.java

```diff
@@ -0,0 +1,44 @@
+package Model;
+
+import java.util.ArrayList;
+
+public class GameModel {
+    ArrayList<Car> cars = new ArrayList<>();
+
+    public GameModel(){}
+
+    public void produceCar(String names, int count){
```

Comment:
자동차를 실행한다는 것보다 자동차 게임을 실행한다는 이름이 더 좋지 않을까요?
```suggestion
    public void produceCarGame(String names, int count){
```